src/modules.d.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: The DCC fork of vpq is not working at the moment
+// declare module '@digitalcredentials/vpqr';
```

Comment:
I enabled Issues on the DCC fork -- do you mind opening an issue with more details?

Reply:
Great, thanks! Here is the issue: https://github.com/digitalcredentials/vpqr/issues/2